other/world.ts

```diff
@@ -105,6 +107,14 @@ export default class World {
     return { x, y };
   }
 
+  private getMaxBotMoveDistance(radius: number): number {
+    const rawSpeed = 1 / Math.pow(radius, this.moveDistanceChangePow);
+    const rawMax = 1 / Math.pow(this.newBotRadius, this.moveDistanceChangePow);
+
+    const normalizedSpeed = (rawSpeed / rawMax) * this.maxMoveDistance;
```

Comment:
Unless I am missing something, isn't the code below doing the same, but express it more clearly?

```suggestion
    const speedRatio = Math.pow(this.newBotRadius / radius, this.moveDistanceChangePow);
    const normalizedSpeed = speedRatio * this.maxMoveDistance;
```